blockchains/ethereum/tokenlist.json

```diff
@@ -3168,7 +3168,7 @@
             "type": "ERC20",
             "address": "0xdd974D5C2e2928deA5F71b9825b8b646686BD200",
             "name": "Kyber Network Crystal",
```

Comment:
@leedaniil can you delete `KNCL` ? because the token is already migrated to the new token.

Reply:
I think you should add a check to check status of the token, if not active - throw an error

Reply:
okay, I marked it as abandoned